pkgs/development/python-modules/splunk-sdk/default.nix

```
@@ -0,0 +1,29 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "splunk-sdk";
+  version = "2.1.0";
+
+  src = fetchFromGitHub {
+    owner = "splunk";
+    repo = "splunk-sdk-python";
+    rev = "refs/tags/${version}";
+    hash = "sha256-N+QQ4DSkx7yakROhcJ2ISXPWFa7BXDeSUULNquhDPrg=";
+  };
+
+  build-system = [ setuptools ];
+
```

```suggestion
  dependencies = [ deprecation ];
```
https://github.com/splunk/splunk-sdk-python/blob/d36db5ee76b3ba1837ac0157e7386fd17a63ff7f/setup.py#L37